src/Api/User/auth/index.js

```diff
@@ -37,7 +37,7 @@ const useAuth = () => {
         return enqueueSnackbar(res.data.message, { variant: 'error' });
       } 
       setCookies("access_token", res.data.token);
-      window.localStorage.setItem('user',JSON.stringify(res.data.user));
+      window.localStorage.setItem('userId',JSON.stringify(res.data.userId));
```

Comment:
Extract the logic of local storage into a service that will do it for you, 

For example : 

```
class A...
```

And then you can use it like this : 

```
A.saveToLocalStorage('key');

```